frontend/mulighetsrommet-veileder-flate/src/components/joyride/Steps.tsx

```diff
@@ -182,13 +190,3 @@ export const opprettAvtaleSteps: MulighetsrommetStep[] = [
     id: "opprett-avtale",
   },
 ];
-
-export const oversiktenLastStep: MulighetsrommetStep[] = [
```

Comment:
Tror denne har blitt automatisk trigget etter at man gjort seg ferdig med `oversiktenSteps`, uavhengig av om man avbrøt eller trykket hele veien gjennom.
Så oppførselen blir vel litt annerledes nå som de er kombinert (dette steget vil ikke lengre dukke opp om man avbryter joyride) - helt greit for meg, tenkte bare jeg skulle nevne siden det var litt uklart om det var en bevisst endring eller ikke :blush:

Reply:
Ja, Tania og jeg diskuterte denne i går. Vi bestemte oss for å slå sammen dette siste steget i oversikten istedenfor, og så fjerne `oversiktenLastStep` for å forenkle litt.